Tradução do submit-to-creator-showcase (#33)
gatsbyjs/gatsby-pt-BR: docs/contributing/submit-to-creator-showcase.md

---

> title: Submeter ao Mostruário de Criadores

Reviewer: Traduzi a página submit-to-plugin-library e utilizei a palavra "enviar" ao invés de "submeter". O que acha de usarmos "enviar"?

> title: Submeter ao Mostruário de Criadores

Reviewer: Acho que "showcase" não deve ser traduzido. No WordPress.org, por exemplo, essa palavra não é traduzida.

Author: faz sentido 🤔

> title: Submeter ao Mostruário de Criadores

Author: Não me parecem ser a mesma coisa: https://www.linguee.com.br/ingles-portugues/traducao/showcase.html

---

Quer fazer parte do [Mostruário de Criadores?](/creators)? Siga essas instruções.

## Passos

Há apenas dois passos :)

> 1.  Se é sua primeira contribuição para o repositório _open source_ do Gatsby, siga as [Orientaçòes de Contribuição](/contributing/code-contributions/).

Reviewer 2: Se é sua -> Se esta é sua
Orientaçòes -> Orientações

> 1.  Se esta é sua primeira contribuição para o repositório _open source_ do Gatsby, siga as [Orientações de Contribuição](/contributing/code-contributions/).

Reviewer 3: Acho que poderia ser interessante deixar desta forma por soar melhor e facilitar a leitura.
```suggestion
1.  Se essa é sua primeira contribuição para o repositório _open source_ do Gatsby, siga as [orientações de contribuição](/contributing/code-contributions/).
```

Reviewer 4: Acho que o correto seria `esta` mesmo

> 2.  Envie uma foto sua or um logo da sua empresa/agência para [`esse diretório`](https://github.com/gatsbyjs/gatsby/tree/master/docs/creators/images). Imagens devem ter um aspecto de proporção quadrado com o mínimo de 500px. (ex: 500px X 500px) até o máximo de resolução de 1000px e deve ter o mesmo nome que você colocar no campo nome em creators.yml, porém com hífens ao invés de espaços.

Reviewer 2: or um logo da sua -> ou um logo da sua

A frase após isso exige uma reformulada, já que ficou muito literal. Sugiro: "Imagens devem ter um aspecto quadrado com relsolução mínima de 500px e máxima de 1000px. A imagem deve possuir o mesmo nomê que você colocar no campo nome em `creators.yml`, substituindo os espaços por hífens."

Author: Reformulei e peguei a maioria das suas sugestões

> 2.  Envie uma foto sua ou um logo da sua empresa/agência para [`esse diretório`](https://github.com/gatsbyjs/gatsby/tree/master/docs/creators/images). Imagens devem ter um aspecto quadrado de proporção com uma resolução mínima de 500px (ex: 500px X 500px) e máxima de 1000px. A imagem deve possuir o mesmo nome que você colocar no campo "nome" em creators.yml, substituindo espaços por hífens.

Author: upload precisaria ser traduzido, não? Acha que ficou confuso o "Envie uma foto"?

> 2.  Envie uma foto sua ou um logo da sua empresa/agência para [`esse diretório`](https://github.com/gatsbyjs/gatsby/tree/master/docs/creators/images). Imagens devem ter um aspecto quadrado de proporção com uma resolução mínima de 500px (ex: 500px X 500px) e máxima de 1000px. A imagem deve possuir o mesmo nome que você colocar no campo "nome" em creators.yml, substituindo espaços por hífens.

Reviewer 5: Na minha opinião neste contexto ficou boa a tradução do @luizcieslak, tem o mesmo sentido com uma linguagem mais cotidiana.

    Por exemplo,

    **se o nome é:** _Fabian Schultz_

    **o nome da imagem deve ser,** _fabian-schultz.jpg_

    **se o nome é:** _Iron Cove Solutions_

    **o nome da imagem deve ser,** _iron-cove-solutions.jpg_

3.  Edite o arquivo [`creators.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/creators/creators.yml) adicionando sua submissão para o fim da lista de sites no seguinte formato:

> ```yaml:title=docs/creators/creators.yml

Reviewer 2: Gostaria de uma sugestão. Devemos manter isso em inglês ou em português? Como temos que preencher em inglês no repositório core, imagino que faça sentido manter em inglês.
Alguem tem alguma outra opinião?

Author: Em inglês mesmo pois pode causar um preenchimento incorreto de quem lê

Reviewer 6: Substituir "para o fim" por "ao fim" (formal) ou "no fim" (coloquial)

  github: "https://github.com/githubusername"
  image: images/image.jpg

  # Agora, você pode escolher true apenas para `for_hire` ou para `hiring`, mas não para ambos.
  for_hire: true
  hiring: false

>   # Se você marcar `portfolio: true`, qualquer site que você tem no Mostruário de Sites será mostrado com `built_by: [imagine seu nome aqui]` será relacionado ao seu Perfil de Criador. Então certifique que o `name` em `creators.yml` é exatamente o mesmo em `built_by` em `sites.yml`.

Reviewer 4: ```suggestion
  # Se você marcar `portfolio: true`, qualquer site que você tem no Mostruário de Sites será mostrado com `built_by: [imagine seu nome aqui]` será relacionado ao seu Perfil de Criador. Então certifique que o `name` em `creators.yml` é exatamente o mesmo de `built_by` em `sites.yml`.
```

  portfolio: true
```

> Use o seguinte _template_ para garantir que os campos obrigatórios estão inseridos:

Reviewer: "template" poderia ser traduzido como "modelo"

Author: Fiz um commit pra arrumar isso

> Use o seguinte _template_ para garantir que os campos obrigatórios estão inseridos:

Author: Fiz um commit pra arrumar isso

  portfolio: (optional)
```

> 4. Se você enviou seus websites para o Mostruário antes mas não preencheu o campo "built_by", você deve editar o arquivo [`sites.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml) e adicionar seu nome (e o campo built_by se ele não estiver presente) lá também para garantir que as peças do seu portifólio estão ligadas à sua página.

Reviewer 2: ```suggestion
4. Se você enviou seus websites para o Mostruário antes mas não preencheu o campo "built_by", você deve editar o arquivo [`sites.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml) e adicionar seu nome (e o campo `built_by` se ele não estiver presente) para garantir que as peças do seu portifólio estejam ligadas à sua página.
```

### Processo de revisão

> Por padrão, todas as edições enviadas ao Mostruário de Criadores será revisada pelo processo regular de aprovação de _PR_ e processos de _merge_.

Reviewer 4: ```suggestion
Por padrão, todas as edições enviadas ao Mostruário de Criadores serão revisadas pelos processos regulares de aprovação de _PR_ e de _merge_.
```

> Por padráo, todas as edições enviadas ao Mostruário de Criadores será revisada pelo processo regular de aprovação de _PR_ e processos de _merge_.

Reviewer 2: padráo -> padrão

### Mudou de ideia / precisa editar sua submissão?

> Se você precisa editar alguma coisa na sua submissão posteriormente, simplesmente edite o arquivo `creators.yml` submetendo outro PR.

Author: Fiz um commit pra arrumar isso
